src/main.cpp

```diff
@@ -3356,6 +3397,10 @@ bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& sta
     if (block.GetBlockTime() <= pindexPrev->GetMedianTimePast())
         return state.Invalid(false, REJECT_INVALID, "time-too-old", "block's timestamp is too early");
 
+    // Check timestamp
```

Comment:
To prevent yet more includes and the crazy-looking GetAdjustedTime call from within the GetBlock stuff. See https://github.com/bitcoin/bitcoin/pull/8068#issuecomment-224781229